packages/interfaces/src/filter.ts

```diff
@@ -12,24 +14,33 @@ export type SubscribeOptions = {
   keepAlive?: number;
 };
 
-export interface IFilterSubscription {
+export type IFilter = IReceiver & IBaseProtocolCore;
+
+export interface ISubscriptionSDK {
```

Comment:
why not simply `ISubscription`? 

overall I think we should stop abusing `SDK` naming as it doesn't give any useful information to consumer (we expected only super advanced devs use `core`)

Reply:
we currently follow `..SDK` naming across all protocol implementations. Creating an issue to reflect on the current naming scheme: https://github.com/waku-org/js-waku/issues/1998

Reply:
thanks, let's follow up